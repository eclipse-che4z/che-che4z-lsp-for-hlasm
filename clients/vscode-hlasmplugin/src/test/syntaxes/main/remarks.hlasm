         L     0,0                     remark
         L     0,0                     (remark)
         L     0,0                     'remark'
