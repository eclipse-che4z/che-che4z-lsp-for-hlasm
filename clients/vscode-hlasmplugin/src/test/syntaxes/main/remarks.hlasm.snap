>         L     0,0                     remark
#^^^^^^^^^ source.hlasm
#         ^ source.hlasm entity.name.function.hlasm
#          ^^^^^ source.hlasm
#               ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#                ^ source.hlasm
#                 ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#                  ^ source.hlasm
#                   ^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
>         L     0,0                     (remark)
#^^^^^^^^^ source.hlasm
#         ^ source.hlasm entity.name.function.hlasm
#          ^^^^^ source.hlasm
#               ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#                ^ source.hlasm
#                 ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#                  ^ source.hlasm
#                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
>         L     0,0                     'remark'
#^^^^^^^^^ source.hlasm
#         ^ source.hlasm entity.name.function.hlasm
#          ^^^^^ source.hlasm
#               ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#                ^ source.hlasm
#                 ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#                  ^ source.hlasm
#                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm comment.hlasm
>